### DISCUSSION TITLE
Raycast functionality?

### DISCUSSION THREAD

**Discussion by minqi** [google/brax#266] [General]
Is there any equivalent of Mujoco's raycast feature in Brax or planned for a future release? I am interested in creating an environment that provides the agent with LIDAR-style observations.

**Comment by btaba** [comment]
Hi @minqi , we don't currently have that implemented and don't have that specific feature planned as of now. The next closest thing: brax/io/image.py (which we do plan on overhauling, which may include convenient raycasting as a byproduct)
